README.md

```diff
@@ -320,12 +320,12 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
   ilab model download
   ```
 
-  `ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace and store it in a `models` directory:
+  `ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace to the `~/.cache/instructlab/models` directory:
 
   ```shell
   (venv) $ ilab model download
   Downloading model from instructlab/merlinite-7b-lab-GGUF@main to models...
-  (venv) $ ls models
+  (venv) $ ls ~/.cache/instructlab/models/
```

Comment:
We're using `$HOME` instead of `~` everywhere else in the README

Reply:
This is actually not true for platforms that are not XDG-compatible. (macosx or windows?) Should we document using ~/.cache (and ~/.local) when it's not universal?

---

btw I don't have a great alternative, except maybe adding commands to `ilab` itself to list each of the used directories and then use the output of the commands as part of the commands. (e.g. you would have `ilab config directory download` returning the current platform download dir; then you'd instruct people to run `ls $(ilab config directory download)`.

---

Another alternative specific to this section of the README is - use `ilab model list` to list downloaded models. It should not matter as much to users (until it does!) where exactly the downloaded model is stored, as long as it's "seen" by the tool itself.

Reply:
FYI posted this suggestion to introduce `config dir` commands here: https://github.com/instructlab/instructlab/issues/1977 that would allow us to `ls $(ilab config dir downloads)` here.